resources/shaders/bloom_vertical.hlsl

```diff
@@ -0,0 +1,33 @@
+#include "dof_util.hlsl"
+#include "bloom_util.hlsl"
+
+Texture2D source : register(t0);
+RWTexture2D<float4> output : register(u0);
+SamplerState s0 : register(s0);
+
+[numthreads(16, 16, 1)]
+void main_cs(int3 dispatch_thread_id : SV_DispatchThreadID)
```

Comment:
Entire file is duplicated with bloom horizontal. create 1 pass with a variable for horizontal or vertical.